src/FixedDataTableBufferedRows.js

```diff
@@ -50,7 +50,6 @@ var FixedDataTableBufferedRows = React.createClass({
   },
 
   componentDidMount() {
-    setTimeout(this._updateBuffer, 1000);
```

Comment:
It doesn't exist - if you're looking at https://github.com/schrodinger/fixed-data-table-2/search?utf8=%E2%9C%93&q=_updateBuffer&type= , you can see it was last indexed almost a month ago. This was the only usage left, and also even if the indexing was correct, you can see that _updateBuffer was not defined anywhere.